openml/extensions/sklearn/extension.py

```diff
@@ -1112,8 +1176,11 @@ def _prediction_to_probabilities(
         # but not desirable if we want to upload to OpenML).
 
         model_copy = sklearn.base.clone(model, safe=True)
+        # security check
```

Comment:
'Security Check' is too vague. Something explicit like 'prohibit users from optimizing `n_jobs`' seems better to me.

Reply:
But if you think that's too explicit I still think I would prefer 'Sanity check' over 'Security Check'.